app/javascript/mastodon/locales/pt-BR.json

```diff
@@ -162,6 +162,8 @@
   "confirmations.discard_edit_media.message": "Há mudanças não salvas na descrição ou pré-visualização da mídia. Descartar assim mesmo?",
   "confirmations.domain_block.confirm": "Bloquear instância",
   "confirmations.domain_block.message": "Você tem certeza de que deseja bloquear tudo de {domain}? Você não verá mais o conteúdo desta instância em nenhuma linha do tempo pública ou nas suas notificações. Seus seguidores desta instância serão removidos.",
+  "confirmations.edit.confirm": "Editar",
+  "confirmations.edit.message": "Editar agora sobrescreverá o toot que você está compondo. Deseja continuar?",
```

Comment:
Looks good overall, but changing translation files directly runs a risk of creating merge conflicts with the automated Crowdin PRs. It's safer to change only `en.json` in PRs and then change the translations in Crowdin.

Reply:
Ok I'm fixing this